jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/PluginConfigurationProcessor.java

```diff
@@ -68,12 +74,25 @@ static AbsoluteUnixPath getAppRootChecked(JibExtension jibExtension) {
    * @throws InvalidImageReferenceException if parsing the base image configuration fails
    */
   static PluginConfigurationProcessor processCommonConfiguration(
-      JibLogger logger, JibExtension jibExtension, GradleProjectProperties projectProperties)
+      Logger logger, JibExtension jibExtension, GradleProjectProperties projectProperties)
       throws InvalidImageReferenceException, NumberFormatException {
     JibSystemProperties.checkHttpTimeoutProperty();
 
     // TODO: Instead of disabling logging, have authentication credentials be provided
-    GradleJibLogger.disableHttpLogging();
+    // Disables Apache HTTP client logging.
+    OutputEventListenerBackedLoggerContext context =
```

Comment:
Unrelated, but looking at this, I think this `processCommonConfiguration` is not being called for exporting a Docker context, meaning it's not disabled for Docker context export?